.travis.yml

```diff
@@ -4,7 +4,6 @@ env:
   - UPGRADE_WEBPY=false
   - UPGRADE_WEBPY=true
 python:
-  - 2.7
```

Comment:
I think we'll want to be compliant with both here

Reply:
Yes.  2to3 often breaks Python 2 compatibility to achieve Python 3 compatibility.  It is better to use futurize or modernize instead.  These are wrappers around 2to3 that curb its most destructive tendencies.